src/tools.cpp

```diff
@@ -254,6 +252,9 @@ void replaceStringInPlace(std::string& subject,
                           const std::string& search,
                           const std::string& replace)
 {
+  if (search.empty())
```

Comment:
If the search is empty if means that we replace nothing. It should return the original string here.

Reply:
True. Sorry for the confusion.